dlt/common/destination/client.py

```diff
@@ -539,6 +539,37 @@ def update_stored_schema(
             )
         return expected_update
 
+    def update_stored_schema_destructively(
```

Comment:
this is overall a good idea to add to JobClient but we should be more selective. Look at my top level review. We need 1. mixin class 2. it needs to be optional